src/controllers/attachment/index.ts

```diff
@@ -102,11 +103,14 @@ export class attachment extends Controller {
 
     if (!req.body && !file) throw new BadRequest('nothing to upload')
 
-    const [{ id, filename, binary_blob, created_at }]: any[] = await this.db
+    const buffer = Buffer.from(file?.buffer || JSON.stringify(req.body))
+    if (buffer.length > env.FILE_UPLOAD_MAX_SIZE) throw new BadRequest(`Over size limit: ${env.FILE_UPLOAD_MAX_SIZE}`)
```

Comment:
seeing `FILE_UPLOAD_MAX_SIZE` in `dscp-api` made me realise we weren't limiting it in this API